modules/bot/controllers/privates/Admin_filter_whitelistController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace app\modules\bot\controllers\privates;
+
+use Yii;
+use \app\modules\bot\components\response\SendMessageCommand;
+use \app\modules\bot\components\response\EditMessageTextCommand;
+use TelegramBot\Api\Types\Inline\InlineKeyboardMarkup;
+use app\modules\bot\components\Controller as Controller;
+use app\modules\bot\models\Chat;
+use app\modules\bot\models\Phrase;
+
+/**
+ * Class FilterChatController
+ *
+ * @package app\controllers\bot
+ */
+class Admin_filter_whitelistController extends Controller
+{
+    /**
+     * @return array
+     */
+    public function actionIndex($groupId = null)
+    {
+        $telegramUser = $this->getTelegramUser();
+        $telegramUser->getState()->setName(null);
+        $telegramUser->save();
+
+        $groupTitle = Chat::find()->where(['id' => $groupId])->one()->title;
```

Comment:
Please, make the following changes:
- Add a check that chat with specified `$groupId` exists.
- Rename variable `$groupId` to `$chatId`. Don't forget to edit route in config